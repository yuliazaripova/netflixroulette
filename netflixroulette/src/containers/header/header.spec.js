import * as React from 'react';

import Header from './header';

describe("Header component", () => {
  it("should render Header component without Btn", () => {
    const component = shallow(<Header/>);
    expect(component).toMatchSnapshot();
  })
  it("should render Header component with Btn", () => {
    const component = shallow(<Header renderBtn={true}/>);
    expect(component).toMatchSnapshot();
  })

})