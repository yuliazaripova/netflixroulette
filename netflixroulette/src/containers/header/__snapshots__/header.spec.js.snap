// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header component with Btn 1`] = `
<header>
  <Logo />
  <BackBtn />
</header>
`;

exports[`Header component should render Header component without Btn 1`] = `
<header>
  <Logo />
</header>
`;
